Snowflake.Data/Client/SnowflakeDbConnection.cs

```diff
@@ -321,17 +321,17 @@ public override Task OpenAsync(CancellationToken cancellationToken)
                 cancellationToken);
         }
 
-        public Mutex GetArrayBindingMutex()
+        internal Mutex GetArrayBindingMutex()

```

Comment:
Technically, it's a breaking change @sfc-gh-ext-simba-hx 
I would consider grouping all such noticed things into a separate PR and merge it once we will be having more unavoidable BCR changes. Otherwise let's revert it. If we need it for some reason  we must start a BCR process.
cc @sfc-gh-igarish

Reply:
done